content.js

```diff
@@ -10,98 +10,60 @@ var Post = class Post {
     }
 };
 
-function loopWhole(i,len, idp, list) {
-    for (i ; i < len - 1; i++) {
-        if (document.querySelector("#pagelet_group_mall").querySelector("div").querySelectorAll("._4-u2, .mbm, ._4mrt, ._5jmm, ._5pat, ._5v3q, ._7cqq, ._4-u8")[i].getAttribute('id') === idp) {
+function getData(idp, list) {
+
+    let len = document.querySelectorAll(".userContentWrapper").length;
+
+    for (let i = 0; i < len; i++) {
+        let main = document.querySelector("._5pcb");
+        let postContent = document.querySelector('.userContentWrapper');
+        if (main.querySelector('._4mrt').getAttribute('id') === idp) {
```

Comment:
I think that this if statement should be on the top function block, and no inside for